lib/audit/moab_to_catalog.rb

```diff
@@ -39,14 +39,26 @@ def self.seed_catalog(storage_dir)
   # Shameless green. In order to run several seed "jobs" in parallel, we would have to refactor.
   def seed_from_disk
     Settings.moab.storage_roots.each do |storage_root|
+      start_msg = "#{Time.now.utc.iso8601} Seeding starting for #{storage_root}"
```

Comment:
it was a tossup whether to put this here, or in .seed_catalog.   Happy to hear a good reason to choose one vs. the other.